package.json

```diff
@@ -128,6 +129,7 @@
     "prettier": "1.2.2",
     "purify-css": "1.2.5",
     "raml-validator-loader": "0.1.11",
+    "react-addons-test-utils": "15.6.2",
```

Comment:
We need this as long as we are under React 15 because of `enzyme-adapter-react-15.4`